### PR TITLE
[13.0][MIG] project_status

### PR DESCRIPTION
**Pull request by bodedra**

### CODE REVIEW

**Comment by bodedra** [comment]
@dreispt This PR is ready for review. I have tested on runbot and works fine.

**Review by alexey-pelykh — APPROVED** [review]
Code seems LGTM

File: project_status/__manifest__.py [hunk 1]
```diff
@@ -1,28 +1,22 @@
...
-    'website': "https://github.com/OCA/project",
-
-    'category': 'Project Management',
-    'version': '12.0.1.1.1',
-    'license': 'AGPL-3',
-
+    "author": "Patrick Wilson, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/project",
+    "category": "Project Management",
+    "version": "13.0.1.1.1",
```
Comment (alexey-pelykh) [inline project_status/__manifest__.py]: Nope, it's not, don't mix reformatting and version change. On top of that, version is not correct

**Review by alexey-pelykh — CHANGES_REQUESTED** [review]

File: project_status/__manifest__.py (hunk 1 above)
Comment (bodedra) [reply project_status/__manifest__.py]: I remembered that when I run pre-commit command, it failed. So I changed version from 12 to 13 and it passed.

Comment (bodedra) [reply project_status/__manifest__.py]: Will ocabot merge command will take care it?

Comment (alexey-pelykh) [reply project_status/__manifest__.py]: No it won't. The [MIG] commit should contain change 12.0.1.1.1 => 13.0.1.0.0

Comment (alexey-pelykh) [reply project_status/__manifest__.py]: And the reformat commit should not have a version change at all

Comment (bodedra) [reply project_status/__manifest__.py]: Let me try it one more time.

Comment (bodedra) [reply project_status/__manifest__.py]: Comment addressed!

**Comment by bodedra** [comment]
@alexey-pelykh Comment addressed.

**Review by alexey-pelykh — APPROVED** [review]

**Comment by dreispt** [comment]
TravisCI tests are missing.
@bodedra Can you rebase against current 13.0 to see if that makes TravisCI build?
Thanks

**Comment by bodedra** [comment]
@dreispt CI is :green_apple: 

I have tested on runbot and it works fine.

http://3418061-662-0d30f8.runbot2.odoo-community.org/web#action=192&model=project.status&view_type=list&cids=&menu_id=98

**Comment by bodedra** [comment]
@dreispt Would you please remove "WIP" label. It is ready for review.

**Review by Saran440 — APPROVED** [review]
Functional Test :+1:

**Review by HaraldPanten — APPROVED** [review]
Functional review

**Comment by bodedra** [comment]
@OCA/project-service-maintainers Can we merge this?

**Comment by alexey-pelykh** [comment]
@bodedra please squash the commits as per https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0

**Comment by bodedra** [comment]
@alexey-pelykh It's already taken care.

Firefox_Screenshot_2020-05-11T07-17-14 075Z

**Comment by alexey-pelykh** [comment]
@bodedra no it's not :)

> Squash administrative commits (if any) with the previous commit for reducing commit noise. They are named as "[UPD] README.rst", "[UPD] Update $MODULE.pot", "Update translation files" and similar names, and comes from OCA-git-bot, oca-travis or oca-transbot.

image

**Comment by bodedra** [comment]
@alexey-pelykh Done.

**Pull request merged**